### PR TITLE
Security Control: CM-07(02)

### PR DESCRIPTION
**Pull request by carltonsmith**
### Summary of Changes
Adds security control implementation statement
Addresses issue #811

### CODE REVIEW

**Comment by lfrohlich** [comment]
@carltonsmith thanks for re-opening this for me. Could you rename it "Security Control: CM-07(02)" and make sure the path would put it in the right security-control docs folder? I'm not sure how to do that 😬

**Comment by carltonsmith** [comment]
> @carltonsmith thanks for re-opening this for me. Could you rename it "Security Control: CM-07(02)" and make sure the path would put it in the right security-control docs folder? I'm not sure how to do that 😬

All set!

**Comment by lfrohlich** [comment]
@tdrammeh1 and @PenyinAsare, this is ready for your review.

**Comment by lfrohlich** [comment]
Per our discussion 4/7/21 we will add a line about the clamav

File: docs/Security-Compliance/Security-Controls/cm-7-2/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+CM-7 (2) Additional FedRAMP Requirements and Guidance:  
+Guidance: This control shall be implemented in a technical manner on the information system to only allow programs to run that adhere to the policy (i.e., white listing).  This control is not to be based off of strictly written policy on what is allowed or not allowed to run.  
+
+### TDP Implementation
+ACF reviews the TDP cloud.gov environments at least annually to identify unnecessary and/or nonsecure functions, ports, protocols, and services; and disables functions, ports, protocols, and/or services deemed to be unnecessary and/or unsecure within the information system.  Please refer to the [architecture decision record](https://github.com/HHS/TANF-app/tree/main/docs/Architecture%20Decision%20Record), [security compliance documents](https://github.com/HHS/TANF-app/tree/main/docs/Security-Compliance), and other [technical documentation](https://github.com/HHS/TANF-app/tree/main/docs/Technical-Documentation) for additional implementation information.
+
```
Comment (ADPennington) [inline docs/Security-Compliance/Security-Controls/cm-7-2/index.md:11]:
```suggestion
[Architectural decision record 012](https://github.com/HHS/TANF-app/blob/main/docs/Architecture%20Decision%20Record/012-antivirus-strategy.md) captures information regarding the TDP virus scanning process and includes a description for how anti-virus definitions will be kept up to date automatically.
```

**Review by ADPennington — COMMENTED** [review]
@lfrohlich added suggested language from #832

**Review by lfrohlich — APPROVED** [review]
Thanks for adding!

**Comment by lfrohlich** [comment]
@tdrammeh1 and @PenyinAsare, this is ready for your review.

**Comment by PenyinAsare** [comment]
Approved

**Pull request merged**